platform/functionality/resolve/resolve.go

```diff
@@ -23,16 +24,28 @@ func HandleResolve(pattern string, sr schema.Registry, ir elasticsearch.IndexRes
 		sourcesToShow = &sourcesFromElasticsearch
 	}
 
+	var filteredOut []elasticsearch.Index
```

Comment:
```suggestion
	var filtered []elasticsearch.Index
```
`filteredOut` is a little confusing, because this array contains items that are actually _kept_